enterprise/api/v1/views.py

```diff
@@ -63,6 +62,7 @@ class EnterpriseModelViewSet(EnterpriseViewSet):
     """
 
     filter_backends = (filters.OrderingFilter, filters.DjangoFilterBackend)
```

Comment:
We should add `permissions_classes=(DjangoModelPermissions, )` here.

Reply:
:+1:

Reply:
Actually this would get overwritten below, so placing it in both `EnterpriseReadOnlyModelViewSet` and `EnterpriseReadWriteModelViewSet`.